src/raft/raft.rs

```diff
@@ -207,13 +207,13 @@ pub struct Raft<T: Storage> {
     pub check_quorum: bool,
     pre_vote: bool,
 
-    heartbeat_timeout: usize,
+    pub heartbeat_timeout: usize,
```

Comment:
We also have a public `get_heartbeat_timeout` function.

Btw, you can add a public `get_randomized_election_timeout` too.